### ISSUE TITLE
Exception when connecting with RedisNativeClient in .NET Core

### ISSUE THREAD

**Issue by dabide** [ServiceStack/Issues#514]
RedisNativeClient throws an `ArgumentNullException` when connecting in .NET Core:

```
System.ArgumentNullException: Value cannot be null.
Parameter name: enumType
   at System.Enum.TryParseEnum(Type enumType, String value, Boolean ignoreCase, EnumResult& parseResult)
   at System.Enum.Parse(Type enumType, String value, Boolean ignoreCase)
   at ServiceStack.Redis.RedisNativeClient.Connect() in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisNativeClient_Utils.cs:line 126
   at ServiceStack.Redis.RedisNativeClient.AssertConnectedSocket() in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisNativeClient_Utils.cs:line 249
   at ServiceStack.Redis.RedisNativeClient.AssertServerVersionNumber() in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisNativeClient_Utils.cs:line 47
   at ServiceStack.Redis.RedisClient.GetServerRole() in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisClient.cs:line 1092
   at ServiceStack.Redis.RedisResolver.CreateRedisClient(RedisEndpoint config, Boolean master) in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisResolver.cs:line 84
   at ServiceStack.Redis.RedisManagerPool.GetClient() in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisManagerPool.cs:line 194
   at ServiceStack.Redis.RedisClientManagerCacheClient.SetAll[T](IDictionary`2 values) in /opt/lib/teamcity-agent/work/8dcfdcec54d0f21a/src/ServiceStack.Redis/RedisClientManagerCacheClient.cs:line 174
```

The offending lines seem to be these:
```
var policyType = AssemblyUtils.FindType("System.Net.Security.EncryptionPolicy");
var policyValue = Enum.Parse(policyType, "RequireEncryption");
```

`policyType` is null.

**Comment by mythz** [comment]
Can you provide the code that causes this Exception?

**Comment by dabide** [comment]
```
RedisManagerPool redisManagerPool = new RedisManagerPool(redisConnectionString);
using (ICacheClient cacheClient = redisManagerPool.GetCacheClient())
{
    cacheClient.Set("foo", "bar");
}
```

**Comment by mythz** [comment]
What `redisConnectionString` are you using? please replace any sensitive info with `xxx`

**Comment by dabide** [comment]
`xxx=@xxx.redis.cache.windows.net?ssl=true`

**Comment by dabide** [comment]
... but I have confirmed that `AssemblyUtils.FindType("System.Net.Security.EncryptionPolicy");` returns null.

**Comment by mythz** [comment]
Should now be fixed from v1.0.34+ that you should be able to upgrade to with a `dotnet restore`.
